### ISSUE TITLE
Disable all rules / enable only one rule

### ISSUE THREAD

**Issue by piegamesde** [markdownlint/markdownlint#523]
I have a large corpus of messy markdown to sanitize, and some rules require a lot of manual intervention and fixup. Is there any way for me to run markdownlint with only a specified set of rules? From what I understand is that it always runs all rules by default, so disabling all of them would require an exhaustive enumeration :(

**Comment by piegamesde** [comment]
Whoops, looks like I got the wrong repo. Moved to https://github.com/igorshubovych/markdownlint-cli/issues/542, which should be the correct one this time. Feel free to reopen if this issue also applies to this project

**Comment by nbehrnd** [comment]
@piegamesde Speaking for markdownlint as it is packaged for Linux Debian ([tracker](https://tracker.debian.org/pkg/ruby-mdl)), both man and info page showcase applications of the syntax checker.  To quote the corresponding section `EXAMPLES` by either call of `man markdownlint`, or `man mdl`:

``` bash
EXAMPLES
       List the objectives of rules MD012, MD022, MD031 and MD032:

              mdl -r MD012,MD022,MD031,MD032 -l

       Scrutinize file example.md only by rules MD012, MD022, MD031 and MD032:

              mdl -r MD012,MD022,MD031,MD032 ./example.md

       Scrutinize file example.md by all rules available except MD033 and MD034:

              mdl -r ~MD033,~MD034 ./example.md

       List all rules associated to the tag indentation:

              mdl -t indentation -l

       Scrutinize file example.md by all rules without tag indentation:

              mdl -t ~indentation ./example.md

       By default, markdownlint reports sort the issues by their  rule.   With  GNU
       coreutils, your shell may resort the output by the corresponding line number
       of the file:

              mdl ./example.md | sort -t ":" -k 2 -n

       Recurrent  (personalized)  check  criteria can be saved as a style.  Snippet
       mystyle.rb below showcases the keywords for comments, rules, tags,  as  well
       as the general syntax available:

              all  # initiate with all rules implemented

              rule 'MD013', :line_length => 100, :ignore_code_blocks => true
              exclude_rule 'MD014'

              # tag :blockquote
              exclude_tag :indentation

       Rules a style file activates can be listed by

              mdl -s ./mystyle.rb -l

       A personalized local style is applied during a check by markdownlint by

              mdl -s ./mystyle.rb ./example.md
```

This documentation is available with markdownlint 0.13.0 as packaged for the current testing release of Debian trixie/13 (by now in a stage of freeze to become stable). Since December 2023, it equally is available as a backport to Debian bookworm/12.  With [repology.org](https://repology.org/project/mdl-markdownlint/packages), you can check the availability of 0.13.0 _as a package_ in other Linux distributions for the same functionality (though the content of their man page might differ).
